javascript/stimulus_reflex.js

```diff
@@ -173,8 +173,11 @@ const extendStimulusController = controller => {
         if (!reflex || !reflex.trim().length) element = element.parentElement
       }
 
-      attributeValues(reflex).forEach(reflex =>
-        this.stimulate(reflex.split('->')[1], element)
+      this.stimulate(
+        attributeValues(reflex)
+          .find(reflex => reflex.split('->')[0] === event.type)
+          .split('->')[1],
```

Comment:
Oh, thanks so much for looking into this :heart:

One question (although I have absolutely no use case for this), but would this only call the first reflex of each event type? Is this something that should be supported?

```
data-reflex="click->SomeReflex#blah click->OtherReflex#bleh"
```

🤔

Reply:
Yes, I think so.

I actually just added documentation for multiple reflex events per element **yesterday**. Before that, you had to look at the code or assume it worked - and by the time you got there, it’d been broken for an indeterminate amount of time.

In short, this is not something a lot of folks have been asking for and just being able to support mouseenter+mouseleave is precisely perfect. I have observed an awkward tendency for developers to fall down rabbit holes where the laws of plausible vs worth it get temporarily suspended. 😺

I am happy to tell people that want three click handlers on one element to just use javascript.